tests/validation/cattlevalidationtest/core/test_services_sidekick.py

```diff
@@ -641,6 +641,62 @@ def test_sidekick_services_deactivate_activate(client,  super_client):
     delete_all(client, [env])
 
 
+def test_sidekick_lbactivation_after_linking(client,
+                                             super_client, socat_containers):
+    service_scale = 2
+    port = "7080"
```

Comment:
Can you use a different port ? 7080 is being used by test_sidekick_for_lb test already